Description: Add the Debian version in the version string
 Display the Debian package version in the About menu, making it easy and
 straightforward to identify a Debian build of rEFIt and its exact version.
Author: Julien BLACHE <jblache@debian.org>

Index: refit-0.14/refit/Makefile
===================================================================
--- refit-0.14.orig/refit/Makefile	2010-04-19 18:32:19.764931838 +0200
+++ refit-0.14/refit/Makefile	2010-04-19 18:32:16.559932151 +0200
@@ -15,7 +15,7 @@
   LIBEG = build64
 endif
 
-LOCAL_CPPFLAGS  = -I$(SRCDIR) -I$(SRCDIR)/../include -I$(SRCDIR)/../libeg
+LOCAL_CPPFLAGS  = -I$(SRCDIR) -I$(SRCDIR)/../include -I$(SRCDIR)/../libeg -DDEBIAN_VERSION="L\"$(DEBVER)\""
 LOCAL_LDFLAGS   = -L$(SRCDIR)/../libeg/$(LIBEG)
 LOCAL_LIBS      = -leg
 
Index: refit-0.14/refit/main.c
===================================================================
--- refit-0.14.orig/refit/main.c	2010-04-19 18:32:19.495931342 +0200
+++ refit-0.14/refit/main.c	2010-04-19 18:32:16.559932151 +0200
@@ -82,7 +82,7 @@
 {
     if (AboutMenu.EntryCount == 0) {
         AboutMenu.TitleImage = BuiltinIcon(BUILTIN_ICON_FUNC_ABOUT);
-        AddMenuInfoLine(&AboutMenu, L"rEFIt Version 0.14");
+        AddMenuInfoLine(&AboutMenu, L"rEFIt Version 0.14 " DEBIAN_VERSION);
         AddMenuInfoLine(&AboutMenu, L"");
         AddMenuInfoLine(&AboutMenu, L"Copyright (c) 2006-2010 Christoph Pfisterer");
         AddMenuInfoLine(&AboutMenu, L"Portions Copyright (c) Intel Corporation and others");
